include/tester/db/Database.hpp

```diff
@@ -82,4 +83,6 @@ class Database
     static constexpr auto DATABASE_NAME = "VoIP-Tester-DB";
```

Comment:
`DATABASE_NAME` should not be a constant - it should be a parameter.

since you have a `VTDatabase` class - store it there and pass it as constructor arguments to the `Database` constructor in the initializer list.

and rename to `VT_DATABASE_NAME`

Reply:
Done